### DISCUSSION TITLE
So many failing tests

### DISCUSSION THREAD

**Discussion by vincentl** [finance-quote/finance-quote#354] [General]
These numbers are slightly out of date because of a few recent merges, but it looks like of the 51 modules there are about 28 with "All tests successful" and 23 with at least some failures and most with complete failure.

Do we want to open an issue for each module with a failing test to discuss if the module is defunct or just needs some minor update?

**Comment by bpschuck** [comment]
Negative. I've been going through them one by one slowly over the last few years.
